src/components/Settings/index.tsx

```diff
@@ -33,6 +43,10 @@ export default function SettingsTab({ placeholderSlippage }: { placeholderSlippa
 
   useOnClickOutside(node, open ? toggle : undefined)
 
+  const [ttl, setTtl] = useUserTransactionTTL()
```

Comment:
@sambacha review if by removing the code below, these imports are necessary at all

Reply:
ok